tools/configure/configure.ml

```diff
@@ -12,499 +12,48 @@
-    "<arch> Specifies the architecture";
-  "-natdynlink", arg_bool (fun p natdynlink -> { p with natdynlink }),
-    "(yes|no) Use dynamic loading of native code or not";
-  "-coqide", arg_ide (fun p coqide -> { p with coqide }),
-    "(opt|byte|no) Specifies whether or not to compile CoqIDE";
-  "-nomacintegration", arg_clear (fun p macintegration -> { p with macintegration }),
-    " Do not try to build CoqIDE MacOS integration";
-  "-browser", arg_string_option (fun p browser -> { p with browser }),
-    "<command> Use <command> to open URL %s";
-  "-with-doc", arg_bool (fun p withdoc -> { p with withdoc }),
-    "(yes|no) Compile the documentation or not";
-  "-byte-only", arg_set (fun p byteonly -> { p with byteonly }),
-    " Compiles only bytecode version of Coq";
-  "-nodebug", arg_clear (fun p debug -> { p with debug }),
-    " Do not add debugging information in the Coq executables";
-  "-profiling", arg_set (fun p profile -> { p with profile }),
-    " Add profiling information in the Coq executables";
-  "-annotate", Arg.Unit (fun () -> die "-annotate has been removed. Please use -annot or -bin-annot instead."),
-    " Removed option. Please use -annot or -bin-annot instead";
-  "-annot", arg_set (fun p annot -> { p with annot }),
-    " Dumps ml text annotation files while compiling Coq (e.g. for Tuareg)";
-  "-bin-annot", arg_set (fun p bin_annot -> { p with bin_annot }),
-    " Dumps ml binary annotation files while compiling Coq (e.g. for Merlin)";
-  "-bytecode-compiler", arg_bool (fun p bytecodecompiler -> { p with bytecodecompiler }),
-    "(yes|no) Enable Coq's bytecode reduction machine (VM)";
-  "-native-compiler", arg_native (fun p nativecompiler -> { p with nativecompiler }),
-    "(yes|no|ondemand) Compilation to native code for conversion and normalization
-     yes: -native-compiler option of coqc will default to 'yes', stdlib will be precompiled
-     no: no native compilation available at all
-     ondemand (default): -native-compiler option of coqc will default to 'ondemand', stdlib will not be precompiled";
-  "-coqwebsite", arg_string (fun p coqwebsite -> { p with coqwebsite }),
-    " URL of the coq website";
-  "-force-caml-version", arg_set (fun p force_caml_version -> { p with force_caml_version }),
-    " Force OCaml version";
-  "-force-findlib-version", arg_set (fun p force_findlib_version -> { p with force_findlib_version }),
-    " Force findlib version";
-  "-warn-error", arg_bool (fun p warn_error -> { p with warn_error }),
-    "(yes|no) Make OCaml warnings into errors (default no)";
-  "-camldir", Arg.String (fun _ -> ()),
-    "<dir> Specifies path to 'ocaml' for running configure script";
-  "-profile", arg_profile,
-    Profiles.doc
-]
-
-let parse_args () =
-  Arg.parse
-    args_options
-    (fun s -> raise (Arg.Bad ("Unknown option: "^s)))
-    "Available options for configure are:"
-
-let () = parse_args ()
-
-let prefs = !prefs
+let _verbose = ref false (* for debugging this script *)
```

Comment:
Why the name change? Also, if it was just for debugging, why keep it here?/Is there a cleaner way to keep it around?

Reply:
Should be removed indeed, was used to debug `Utils.run`.